streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -122,6 +128,18 @@ public final void removeAllThreadLevelSensors() {
         }
     }
 
+    public Map<String, String> taskLevelTagMap(final String taskName) {
+        final Map<String, String> tagMap = threadLevelTagMap();
```

Comment:
`streamsMetrics.threadLevelTagMap(TASK_ID_TAG, ALL_TASKS)` is used for a parent sensor that will be removed as part of KIP-444. When the that sensor will be removed also `streamsMetrics.threadLevelTagMap(TASK_ID_TAG, ALL_TASKS)` and the corresponding method will disappear. I would like to leave it for the future PR, since that code is not touched in this PR.